### PR TITLE
Dutch language 2

### PR DESCRIPTION
**Pull request by derkgort**
This adds a few texts to the dutch language yml.

### CODE REVIEW

File: lang/nl.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ nl:
     COMMENTS: "Reacties"
     EDITTHIS: "Bewerk deze post"
     POSTEDBY: "Auteur"
-    POSTEDON: "Aan"
+    POSTEDON: "op"
```
Comment (dhensby) [inline lang/nl.yml]: I think we need to keep the capitalisation ie: it should be "Op"

Comment (derkgort) [inline lang/nl.yml]: That's OK. Thanks

Comment (derkgort) [inline lang/nl.yml]: Do you need a new request?

Comment (dhensby) [inline lang/nl.yml]: You can just amend the commits in this branch and the PR will update

Comment (dhensby) [inline lang/nl.yml]: Note this needs amending in both places you've changed it

Comment (derkgort) [inline lang/nl.yml]: Done

**Comment by tractorcow** [comment]
Instead of merging it in here, perhaps we should consolidate with https://www.transifex.com/projects/p/silverstripe-blog/viewstrings/#nl/master?

**Pull request closed without merging**